madoop/mapreduce.py

```diff
@@ -26,61 +27,62 @@ def mapreduce(input_dir, output_dir, map_exe, reduce_exe):
     output_dir = pathlib.Path(output_dir)
     if output_dir.exists():
         raise MadoopError(f"Output directory already exists: {output_dir}")
-
-    # Create tmp directories, starting with {ouput_dir}/hadooptmp
-    tmpdir = pathlib.Path(output_dir)/"hadooptmp"
-    if tmpdir.is_dir():
-        shutil.rmtree(tmpdir)
-    tmpdir.mkdir(parents=True, exist_ok=False)
-    map_input_dir = tmpdir/'mapper-input'
-    map_output_dir = tmpdir/'mapper-output'
-    group_output_dir = tmpdir/'grouper-output'
-    reduce_output_dir = tmpdir/'reducer-output'
-    map_input_dir.mkdir()
-    map_output_dir.mkdir()
-    group_output_dir.mkdir()
-    reduce_output_dir.mkdir()
-
-    # Copy and rename input files: part-00000, part-00001, etc.
-    input_dir = pathlib.Path(input_dir)
-    num_map = prepare_input_files(input_dir, map_input_dir)
-
-    # Executables must be absolute paths
-    map_exe = pathlib.Path(map_exe).resolve()
-    reduce_exe = pathlib.Path(reduce_exe).resolve()
+    output_dir.mkdir()
```

Comment:
Thanks for the fix @jaredzh !  This is why we have code reviews!!!

Reply:
Of course!